src/components/modals/CustomizeToolbar.tsx

```diff
@@ -149,7 +151,7 @@ const ModalCustomizeToolbar: FC = () => {
     >
       <h1 className='modal-title'>Customize Toolbar</h1>
       <p style={{ marginTop: '-1em', marginBottom: '1em' }}>
-        &lt;{' '}
+        &lt{' '}
```

Comment:
I think this got changed by accident. I fixed it.

<img width="181" src="https://github.com/user-attachments/assets/8d28a39a-b3ca-4480-9d0e-429a8fec7b27">